src/app/parcels/add/page.tsx

```diff
@@ -0,0 +1,18 @@
+import { Metadata } from "next";
+
+import React from "react";
+
+const AddParcels: () => React.ReactElement = () => {
```

Comment:
sorry I just realised I did this in `clients\add\page.tsx` as well but can you change it to `const AddParcels = (): React.ReactElement => { ... }` to be consistent with the other pages? (would appreciate if you change it for the clients/add page to hehe)